### ISSUE TITLE
[BUG] file upload doesn't always correctly update frontend after refresh

### ISSUE THREAD

**Issue by adrianocr** [hayes0724/shopify-packer#50]
**Describe the bug**
Not always, but sometimes packer/themekit uploads a file and refreshes the page as it should, but the refresh doesn't actually load the new updated file or changes. It still contains the old markup and content. Then you manually refresh and the changes are there. I assume this is an issue with the shopify backend taking a little longer than expected to process the file so a solution may be to add a 0.5s delay to refreshing the page?

**Expected behavior**
After themekit uploads a file, the page should refresh like it currently does, but it should load the updated file(s).

**System (please complete the following information):**
 - OS: macOS big sur
 - Node Version: v14.15.3
 - NPM Version: npm@6.14.9
 - Browser: every

**Packer (please complete the following information):**
 - Version: v1.3.4
 - 
**Theme (please complete the following information):**
 - [Shopify Packer theme](https://github.com/hayes0724/packer-blank-theme) [true]
 - Version: latest

**Comment by hayes0724** [comment]
I had previously had a delay for this issue, sometimes it's needed and other times it loads quick enough not to need it. I think this would be a good setting to have in config (an adjustable delay rate)

**Comment by adrianocr** [comment]
@hayes0724 I agree! Maybe an option in the .env file that lets the user stipulate how many ms to delay by

**Comment by hayes0724** [comment]
Fixed in v1.3.6

usage (packer-config.js):
```
module.exports = {
  // Add a reload delay before refreshing browser in ms, Shopify may need several seconds
  'network.reload': 0,
}

```
